2.3.0/Dockerfile

```diff
@@ -28,8 +28,8 @@ RUN set -eux; \
 # Verify that gosu binary works
     gosu nobody true
 
-ARG GPG_KEY=79B03D059E628478FC9F1D8B152CAD0C46E87B61
-ARG DISTRO_NAME=apache-storm-2.2.0
+ARG GPG_KEY=5167DE337E7370373499FC1DA4A672F11B5050C8
```

Comment:
This key is one of the official signing keys: https://downloads.apache.org/storm/KEYS